Tribler/community/multichain/community.py

```diff
@@ -0,0 +1,427 @@
+            self.send_crawl_request(message.candidate)
+
+    def _get_next_total(self, up, down):
+        """
+        Returns the next total numbers of up and down incremented with the current interaction up and down metric.
+        :param up: Up metric for the interaction.
+        :param down: Down metric for the interaction.
+        :return: (total_up (int), total_down (int)
+        """
+        total_up, total_down = self.persistence.get_total(self._public_key)
+        if total_up == total_down == -1:
+            return up, down
+        else:
+            return total_up + up, total_down + down
+
+    def _get_next_sequence_number(self):
+        return self.persistence.get_latest_sequence_number(self._public_key) + 1
+
+    def _get_latest_hash(self):
+        previous_hash = self.persistence.get_latest_hash(self._public_key)
+        return previous_hash if previous_hash else GENESIS_ID
+
+    def unload_community(self):
+        self.logger.debug("Unloading the MultiChain Community.")
+        super(MultiChainCommunity, self).unload_community()
+        # Close the persistence layer
+        self.persistence.close()
+
+    @forceDBThread
+    def on_tunnel_remove(self, subject, changeType, tunnel, stats, candidate):
+        """
+        Handler for the remove event of a tunnel. This function will attempt to create a block for the amounts that
+        were transferred using the tunnel.
+        :param subject:
+        :param changeType:
+        :param tunnel: The tunnel that was removed (closed)
+        :param stats: The statistics regarding this tunnel
+        :param candidate: The dispersy candidate with whom this node has interacted in the tunnel
+        :return:TGliTmFDTFBLOirMUruvuMNO6fVRukZ2mut3a05I38dkdkzkohaqwZlFT24t/1xCug/pVglwArD+YEG4dx47ohoByy5lWWtQwno=
+        """
+        if isinstance(tunnel, Circuit):
+            bytes_up = stats['bytes_up']
+            bytes_down = stats['bytes_down']
+        elif isinstance(tunnel, RelayRoute):
+            bytes_up = stats['bytes_relay_up']
+            bytes_down = stats['bytes_relay_down']
+        elif isinstance(tunnel, TunnelExitSocket):
+            bytes_up = stats['bytes_exit']
+            bytes_down = stats['bytes_enter']
+        else:
+            self.logger.error("Got a tunnel remove event for an object that is not a Circuit, "
+                              "RelayRoute or TunnelExitSocket")
+            raise TypeError("Got a tunnel remove event for an object that is not a Circuit,"
+                            " RelayRoute or TunnelExitSocket")
+
+        if isinstance(bytes_up, int) and isinstance(bytes_down, int):
+            if bytes_up > MEGA_DIVIDER or bytes_down > MEGA_DIVIDER:
+                # Tie breaker to prevent both parties from requesting
+                if self._public_key > candidate.get_member().public_key:
+                    self.schedule_block(candidate, bytes_up, bytes_down)
+                # else:
```

Comment:
The linked commit doesn't remove this commented out code and the code is still there. Do you really want this commented out stuff to stay?